src/libsync/accessmanager.cpp

```diff
@@ -40,6 +42,16 @@ AccessManager::AccessManager(QObject *parent)
     setConfiguration(QNetworkConfiguration());
 #endif
     setCookieJar(new CookieJar);
+
+    connect(this, &AccessManager::sslErrors, this, [this](QNetworkReply *reply, const QList<QSslError> &errors) {
```

Comment:
Can we make sure this slot is run first using some flag on `connect`?

Reply:
Thinking about it, maybe we should just debug log all erased errors. That way, whoever will work with this code in the future, will get notified about ignored errors.